### ISSUE TITLE
[develop] Collector tab - Cannot read property 'token_info' of undefined

### ISSUE THREAD

**Issue by skenaja** [hicetnunc2000/hicetnunc#535]
**Describe the bug**
Testing on `develop` branch as at 3d4e26bbb1bddb32a5191be4b833de0f81c5bf01

Access any item objkt-detail page, collectors tab:

```
TypeError: Cannot read property 'token_info' of undefined
Collectors
src/pages/objkt-display/tabs/collectors.js:53
  50 | <Padding>
  51 |   <OwnerList
  52 |   owners={filtered}
> 53 |   creator={currentObjkt.token_info.creators[0]}
     | ^  54 |   acc={acc}
  55 |   />
  56 | </Padding>
```

Error is caused by 0afea0f9b9583900d6509d38c784d0c1f385eede from #512 having been overwritten in `develop` (maybe there's been a force push recently?) 

That commit is required for dee109f6b049851ffbd4b139aa40896eafa46eff to work correctly.

I'll submit a patch PR to reinstate it shortly

**Comment by skenaja** [comment]
@andrevenancio - actually, it's not clear whether you do want the functionality reinstated as the comments on #512 aren't very clear what you and @crzypatchwork decided to do after the merge...  This bug is a result of a rebase & partial dropping of some of the commits in 512 & a force-push. 

`src/pages/objkt-display/tabs/collectors.js` is still expecting to see context-saved objkt info from 0afea0f9b9583900d6509d38c784d0c1f385eede, but this was part of dropped commit during rebase.

Let me know how you want me to proceed.

**Comment by andrevenancio** [comment]
Hey @skenaja ! Thanks for this. so right now what's on `main` and what's on `develop` is quite different, and I recently had to push code to `main` only for a feature we want to release today, but the code on develop has new stuff that needs further testing.

About #512 I think what crzypatch work didn't like on your PR was the extra html element showing the range. which I agree add's too much "noise" to a minimal design. and from your screenshot I think its easy for users to get a sense of the range since they are displayed in the buy buttons. One thing I've added but can't remember if its on develop or main, its the price sorting. so the price sorting on the collectors now shows from cheap to expensive. So that helps find the right "range" when before that wasn't ordered.

About this in particular, you're right. I think on the feed's we have a util function checking if `token_info` is available and if its not, we filter it out. Just because some really old objects seem to not have it. 

crzypatch work is working on a database, and that will hopefully help us correct some of those objkt that for a reson or another have broken parameters. But rather than the visual amends on your PR, I think from a code point of view you're right. we should check for `token_info` before requesting it. I'll have a look at your PR later today! Thanks, as always super thorough!

**Comment by skenaja** [comment]
Thanks - makes sense! To help when you look at the PR again:

The reason I was setting & getting `token_info` from state was to highlight the creators and current wallet ids on collectors tab (the first 2 screenshots in #512). Here's where things currently stand:

Here are the commits from the PR that are still in `develop`
- Allow addition of current OBJKT to context state e241367
- Make burn information clearer 753c282
- Fix typo c81ead5

These are those that were dropped:
- Sort swaps in ascending price order 3699494
- Display price range on OBJKT details 91e46d6
- Save current OBJKT data in context 0afea0f

This one was partially kept (owner-swaps component dropped, owner-list component & collector tab page kept):
- Highlight creator & user in collection and swap lists dee109f

The part that remains in dee109f depends on both 0afea0f and e241367.
